src/index.js

```diff
@@ -105,6 +106,10 @@ class ServerlessVpcPlugin {
       if ('createNatInstance' in vpcConfig && typeof vpcConfig.createNatInstance === 'boolean') {
         ({ createNatInstance } = vpcConfig);
       }
+
+      if ('exportOutputs' in vpcConfig && typeof vpcConfig.createNatInstance === 'boolean') {
```

Comment:
exportOutputs

Reply:
fixed in https://github.com/smoketurner/serverless-vpc-plugin/pull/89/commits/6e4a867ef5af569c6f8f879dea02e1c0f4e21418